see_profile/__init__.py

```diff
@@ -40,17 +41,22 @@ def __init__(self, get_response=None):
     def load_setting(self, setting, value):
         setattr(self, setting.lower(), value)
 
-    def format_queries(self, queries):
+    def format_queries_and_time_for_logs(self, queries):
         formatted_queries = []
+        total_time = 0
         for query in queries:
             formatted_sql = sqlparse.format(query['sql'], reindent=True,
                                             keyword_case='upper')
 
             formatted_queries.append(
                 "{}:\n{}".format(query['time'], formatted_sql)
             )
+            total_time += float(query['time'])
 
-        return self.query_separator.join(formatted_queries)
+        log_messages = itertools.chain(
```

Comment:
Maybe a nitpick, but I think using chain here is confusing. I think would have written this as:

```
log_messages = [f"\n{len(queries)} Queries\nTotal time for queries: {total_time}"] + formatted_queries
```

Unless i'm misunderstanding things.